modules/tinymce/src/plugins/table/main/ts/actions/ResizeHandler.ts

```diff
@@ -98,16 +94,16 @@ export const getResizeHandler = function (editor: Editor): ResizeHandler {
     const targetElm = e.target;
     if (isTable(targetElm)) {
 
-      const tableHasPercentage = getRawWidth(targetElm).map((w) => percentageBasedSizeRegex.test(w)).getOr(false);
+      const tableHasPercentage = Util.getRawWidth(editor, targetElm).map(Util.isPercentage).getOr(false);
```

Comment:
`map(x).getOr(y)` === `fold(x, () => y)`. 

`map(predicate).getOr(false)` === `exists(predicate)`

Reply:
Not something I did, but I'll fix it.